lib/muffin_man/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module MuffinMan
-  VERSION = "2.0.6"
+  VERSION = "2.0.7"
```

Comment:
I think we can go with minor version update instead of patch version update as its a new non-breaking functionality that we are adding.

Refer: Summary section from here - https://semver.org/
```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
```